packages/web/components/Setup/SetupUsername.tsx

```diff
@@ -59,7 +59,7 @@ export const SetupUsername: React.FC<SetupUsernameProps> = ({
       <Input
         background="dark"
         placeholder="USERNAME"
-        value={username}
+        value={username ?? ''}
```

Comment:
why this change?

Reply:
Going from an undefined variable to a value causes React to complain about switching from an uncontrolled component to a controlled one.